documentation/modules/auxiliary/admin/kerberos/inspect_ticket.md

```diff
@@ -0,0 +1,157 @@
+4. Optional: either `set AES_KEY aes_key_here` or `set NTHASH nthash_here`
+5. Do: `run` to see the contents of the ticket 
+
+## Scenarios
+
+### Inspecting Ticket contents
+
+This action allows you to see the contents of any ccache or kirbi file,
+If you are able to provide the decryption key we can also show the encrypted parts of the tickets.
+
+1. `TICKET_PATH` - The path to the ccache or kirbi file.
+2. `AES_KEY` - (Optional) Only set this if you have the decryption key and it is an AES128 or AES256 key.
+3. `NTHASH` - (Optional) Only set this if you have the decryption key and it is an NTHASH.
+No other options are used in this action.
+
+**Without Key**
+
+```
+msf6 auxiliary(admin/kerberos/inspect_ticket) > run TICKET_PATH=/path/to/ticket
+```
+
+Example Output:
+```
+Primary Principal: Administrator@WINDOMAIN.LOCAL
+Ccache version: 4
+
+Creds: 1
+  Credential[0]:
+    Server: cifs/dc.windomain.local@WINDOMAIN.LOCAL
+    Client: Administrator@WINDOMAIN.LOCAL
+    Ticket etype: 18 (AES256)
+    Key: 3436643936633032656264663030393931323461366635653364393932613763
+    Ticket Length: 978
+    Subkey: false
+    Addresses: 0
+    Authdatas: 0
+    Times:
+      Auth time: 2022-11-21 13:52:00 +0000
+      Start time: 2022-11-21 13:52:00 +0000
+      End time: 2032-11-18 13:52:00 +0000
+      Renew Till: 2032-11-18 13:52:00 +0000
+    Ticket:
+      Ticket Version Number: 5
+      Realm: WINDOMAIN.LOCAL
+      Server Name: cifs/dc.windomain.local
+      Encrypted Ticket Part:
+        Ticket etype: 18 (AES256)
+        Key Version Number: 2
+        Cipher:
+          1YrnB+fhzeLEq+4NUcXvoEsSI29+gwCDg3qjYdb0YHhqx23BhZGOK9rIQ99uXeuLHSapJAanCE9g/PyyKDE1kggrEHfy6cxwsP25exmN2w3NXVm7P0PqMVON2RBp2S11eIdF/Zibhrs7JbaaVw0Hv8GpbpHdFI0l6Xx3Jz+y0bqFsFNEsU8nEW35Z3Oo2xpI/xTwNTyG1Bmg+bktSLyI6nEPtJXQKcoJTrNhSBNsZ18HZiUPim9EqSCHUh0VbDeLntryh+lt0TIgwhwipHPWnro+Y81dvX5j8ZeBdgKgnoX3jciU629u/RveQJgyw/vLk1KT0RzTbHSwdRk/xi6ghccvew33TKJ8q3nP/JuSWDzaDE6I6v3KgInSZP+XkCAV5VT//U49MtIVIKARcmtXQwVxztMXKlWjIaxQwl9BN6CuyWZjDcafAssjPWgWIAsesmEWHn3btv1BP0a4gvn5f1b7Fu4Gh6w0ARCryxZkSl+6UhJbcdaRT23WhqN24ECGEl0VIX4fuLs6x0gVtAQ2YsI+HkoQYuI+C28gXzJUCac6rJyFQSTsciwj/jVf18ttw1vfGGKa/BVcqscGZoJPpBiuGPBkIbeAOery0Sjn+0tP0tsPYw1OkpzZ7n/j/YdmTX6UAFZjCLbgvF8hoPyider1gntOiSjlLlEUITLTfe5zqWi4gs47Ly6lvggBWW9Yg0fIaPOHYMvsszMLcJz0+dFXtDVI452LIEatLDvp1aKkwGANWYyRgOMlHR3fD030SOTNEb5oa6WigWZQLlhuDbgrfFaWWAMp7opcNbNKy7Iv17EscL7pW2Ygc38VbmbFtdIfvpQ9niwLr2msjzhB7RPihZXcUAlVygLwykq0JDG4fRmoNXzNydbnYlX9E+KW0fHFjoBitAx1xrp9p5Ajwoyy+wIk0mt/aC4pbfcoRjt4GUF/9DhZnH3HiPn4lM9TLMzpiediEtDZtKgGvAAP2cJZn2gsLRlKAtBZvl+ibe1uDzC9g6rnObAx3c+OSG9rmHzBBCq6D8wW6ZjrQy8njNuriC5rnQxUpVhgGvTOkeTphSIHX+D4SuMd+XZ4zqa3DsrHzIeVWAvrTHCDBzy+DKt2RoQTwYmGT+a0YB0btQtgIfRj2OwDtlP65JUxC+/ANelHg73d0REoYistB5ZMmvk=
+```
+
+**With Key**
+```
+msf6 auxiliary(admin/kerberos/inspect_ticket) > run AES_KEY=4b912be0366a6f37f4a7d571bee18b1173d93195ef76f8d1e3e81ef6172ab326 TICKET_PATH=/path/to/ticket
```

Comment:
it's not necessary to have the NTHASH or AES_KEY to run the module properly, how you obtain the key is highly dependant on your environment/workflow
For example if you wish to inspect a ticket you generated with the `forge_ticket` module getting the key is trivial since you used it to generate the ticket
Another potential source is to use the `windows_secrets_dump` module on a system you've already compromised
Because we don't know anything about the  situation the users finds themselves in I don't think it makes a tonne of sense to try and tell the user where they can get a key from

Reply:
After further discussion, we have decided we will likely need to do a more comprehensive documentation effort due to the complex nature of Kerberos and so it may be best to revisit this later along with the other modules. Also this module is typically going to be used by people who already know how to get this information; that being said I do think we should likely revisit this at some point to cater to hobbyists who might be curious about this module and want to know more about how to retrieve this type of info, but this would likely be better served by the comprehensive documentation mentioned previously.